scripts/OverlaySurvey.py

```diff
@@ -285,6 +323,18 @@ def main():
                                 help="output file for the augmented graph")
     parser_augment.set_defaults(func=augment)
 
+    parser_simplify = subparsers.add_parser('simplify',
+                                            help="simplify the master graph "
```

Comment:
We need to explain this command in more detail, right now it's pretty vague. What exactly are we simplifying? Maybe `simplify` isn't the best name for it - we're basically setting up a new graph for the simulation purposes, so perhaps that's what we should name it: `adjustForSimulation` or something like that.

Reply:
Updated the command name, and I also added more explanation to `README.md`.